Packs/Slack/Integrations/SlackV3/SlackV3.py

```diff
@@ -982,7 +986,7 @@ async def slack_loop():
             app_token=APP_TOKEN,
             web_client=ASYNC_CLIENT,
             logger=slack_logger,  # type: ignore
-            auto_reconnect_enabled=False
+            auto_reconnect_enabled=True
```

Comment:
why is this relevant?

Reply:
Did some testing and with the session change we made last time, the sdk will handle the reconnects safely. We didn't have a listener for disconnects so it would just disconnect without raising any errors.